recipes/wx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "wxPython" %}
+{% set version = "4.0.0a2" %}
+{% set sha256 = "3acd5160d69f56b588476d1907eff67b53ad58fb2eba4dedabcdc3490d170338" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [unix]
+
+requirements:
+  build:
+    - python
+    - six
```

Comment:
Could you please try dropping it then? FWIW if it is a build dependency, it normally shows up in `setup_requires` (assuming it has been listed at all).